[2025-02-08T19:02:57.078+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T19:02:57.098+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-19T00:00:00+00:00 [queued]>
[2025-02-08T19:02:57.108+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-19T00:00:00+00:00 [queued]>
[2025-02-08T19:02:57.109+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T19:02:57.125+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_customer_dim_data> on 2024-12-19 00:00:00+00:00
[2025-02-08T19:02:57.140+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1779) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T19:02:57.142+0000] {standard_task_runner.py:72} INFO - Started process 1805 to run task
[2025-02-08T19:02:57.141+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'customer_dim_generator', 'generate_customer_dim_data', 'scheduled__2024-12-19T00:00:00+00:00', '--job-id', '302', '--raw', '--subdir', 'DAGS_FOLDER/customer_dim_gen.py', '--cfg-path', '/tmp/tmpr8im6rhc']
[2025-02-08T19:02:57.144+0000] {standard_task_runner.py:105} INFO - Job 302: Subtask generate_customer_dim_data
[2025-02-08T19:02:57.204+0000] {task_command.py:467} INFO - Running <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-19T00:00:00+00:00 [running]> on host 2c1cd59afcc0
[2025-02-08T19:02:57.313+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='customer_dim_generator' AIRFLOW_CTX_TASK_ID='generate_customer_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-19T00:00:00+00:00'
[2025-02-08T19:02:57.315+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T19:02:57.343+0000] {logging_mixin.py:190} INFO - CSV file './customer_dim_large_data.csv' with 100 rows has been generated successfully.
[2025-02-08T19:02:57.344+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T19:02:57.356+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T19:02:57.357+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=customer_dim_generator, task_id=generate_customer_dim_data, run_id=scheduled__2024-12-19T00:00:00+00:00, execution_date=20241219T000000, start_date=20250208T190257, end_date=20250208T190257
[2025-02-08T19:02:57.415+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T19:02:57.436+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T19:02:57.443+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-08T20:40:37.914+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T20:40:37.927+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-19T00:00:00+00:00 [queued]>
[2025-02-08T20:40:37.938+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-19T00:00:00+00:00 [queued]>
[2025-02-08T20:40:37.939+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T20:40:37.949+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_customer_dim_data> on 2024-12-19 00:00:00+00:00
[2025-02-08T20:40:37.958+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'customer_dim_generator', 'generate_customer_dim_data', 'scheduled__2024-12-19T00:00:00+00:00', '--job-id', '221', '--raw', '--subdir', 'DAGS_FOLDER/customer_dim_gen.py', '--cfg-path', '/tmp/tmpjgi11p08']
[2025-02-08T20:40:37.961+0000] {standard_task_runner.py:105} INFO - Job 221: Subtask generate_customer_dim_data
[2025-02-08T20:40:37.961+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1492) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T20:40:37.963+0000] {standard_task_runner.py:72} INFO - Started process 1506 to run task
[2025-02-08T20:40:38.005+0000] {task_command.py:467} INFO - Running <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-19T00:00:00+00:00 [running]> on host 7e959c1a2a8b
[2025-02-08T20:40:38.068+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='customer_dim_generator' AIRFLOW_CTX_TASK_ID='generate_customer_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-19T00:00:00+00:00'
[2025-02-08T20:40:38.069+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T20:40:38.083+0000] {logging_mixin.py:190} INFO - CSV file './customer_dim_large_data.csv' with 100 rows has been generated successfully.
[2025-02-08T20:40:38.084+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T20:40:38.091+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T20:40:38.091+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=customer_dim_generator, task_id=generate_customer_dim_data, run_id=scheduled__2024-12-19T00:00:00+00:00, execution_date=20241219T000000, start_date=20250208T204037, end_date=20250208T204038
[2025-02-08T20:40:38.139+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T20:40:38.152+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T20:40:38.154+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-08T20:43:25.201+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T20:43:25.219+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-19T00:00:00+00:00 [queued]>
[2025-02-08T20:43:25.228+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-19T00:00:00+00:00 [queued]>
[2025-02-08T20:43:25.228+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T20:43:25.276+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_customer_dim_data> on 2024-12-19 00:00:00+00:00
[2025-02-08T20:43:25.284+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=710) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T20:43:25.285+0000] {standard_task_runner.py:72} INFO - Started process 728 to run task
[2025-02-08T20:43:25.284+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'customer_dim_generator', 'generate_customer_dim_data', 'scheduled__2024-12-19T00:00:00+00:00', '--job-id', '478', '--raw', '--subdir', 'DAGS_FOLDER/customer_dim_gen.py', '--cfg-path', '/tmp/tmpbxjtpm_m']
[2025-02-08T20:43:25.286+0000] {standard_task_runner.py:105} INFO - Job 478: Subtask generate_customer_dim_data
[2025-02-08T20:43:25.333+0000] {task_command.py:467} INFO - Running <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2024-12-19T00:00:00+00:00 [running]> on host 86932746817d
[2025-02-08T20:43:25.412+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='customer_dim_generator' AIRFLOW_CTX_TASK_ID='generate_customer_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-19T00:00:00+00:00'
[2025-02-08T20:43:25.413+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T20:43:25.430+0000] {logging_mixin.py:190} INFO - CSV file './customer_dim_large_data.csv' with 100 rows has been generated successfully.
[2025-02-08T20:43:25.430+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T20:43:25.439+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T20:43:25.440+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=customer_dim_generator, task_id=generate_customer_dim_data, run_id=scheduled__2024-12-19T00:00:00+00:00, execution_date=20241219T000000, start_date=20250208T204325, end_date=20250208T204325
[2025-02-08T20:43:25.459+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T20:43:25.475+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T20:43:25.478+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
